openslides/users/migrations/0004_groups.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.7 on 2016-08-01 14:54
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import Group
+from django.db import migrations
+
+
+def migrate_groups_and_user_permissions(apps, schema_editor):
+    """
+    This function migrates the database to the new groupslogic:
+    - Rename Group 'Anonymus' (pk=1) to 'Default'
+    - Rename Group 'Registered users' (pk=2) to 'oldRegisteres'
```

Comment:
Typo oldRegistered. Maybe we can choose something smarter like "Previous group Registered" or something similar.